### DISCUSSION TITLE
wind information

### DISCUSSION THREAD

**Discussion by aukanai** [jaybo/deepzoom-blog#15] [Q&A] [answered]
Jay,
Really like using your site and present Deepzoom  in any navigation class I teach. Quick question.

I notice when I plan a route and run a simulation the wind arrow on the simulation doesn't seem to align with the NOAA forecast winds. They are close but not the same. Today I ran a simulation from Bellingham to Deception Pass and the simulation showed winds more out of the SE and the NOAA forecast was winds out of the South. Are Deepzoom winds magnetic?

**Comment by jaybo** [comment, marked as answer]
NOAA issues many different forecasts, each with a different purpose, resolution in time and space, and prediction accuracy.  Probably the most widely used is the 0.25 degree resolution worldwide GFS forecast.  This is the source DeepZoom uses for wind forecast display.  This forecast is issued every 6 hours, and then there are variable data processing delays before the information is updated and displayed in DeepZoom.

The North American Mesoscale (NAM) forecast is of shorter duration and is updated only twice a day, but is considered more accurate within the regional bounds of the USA.  Local NOAA forecast offices likely use this as their primary data source and may even further refine forecasts based on local knowledge.

So the differences are likely due to the data source used in each case.  The wind display is **not** using influenced by the local magnetic declination.

**Comment by aukanai** [comment]
As always
Thank you Jay
